pkg/controllers/provisioning/scheduling/nodeclaim.go

```diff
@@ -419,14 +420,18 @@ func filterInstanceTypesByRequirements(instanceTypes []*cloudprovider.InstanceTy
 		// We don't care about the minimum number of instance types that meet our requirements here, we only care if they meet our requirements.
 		_, err.minValuesIncompatibleErr = remaining.SatisfiesMinValues(requirements)
 		if err.minValuesIncompatibleErr != nil {
```

Comment:
We should probably avoid setting the err here if we are supporting relaxation since we technically don't have an error if we had the flexibility to relax it